aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncTimeRegistry.java

```diff
@@ -55,9 +56,11 @@
         });
     }
 
-    <T extends Model> Completable saveLastSyncTime(@NonNull Class<T> modelClazz, SyncTime syncTime) {
+    <T extends Model> Completable saveLastSyncTime(@NonNull Class<T> modelClazz,
```

Comment:
Possibly, consider the same `saveBaseSyncTime(...)` and `saveDeltaSyncTime(...)` to keep the arg count down? (As with the factories on `LastSyncMetadata`.)

Reply:
Followed the same pattern here as well.